CHANGELOG.md

```diff
@@ -43,6 +43,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - All `olmo_core.nn` module configs now inherit from a common base class, `ModuleConfig`.
 - Big changes to `olmo_core.model_ladder` API.
 - Add ngram instance filter to olmo3_ladder.
+- Allow comments in cordoned nodelist for lambda slurm launcher.
```

Comment:
Changelog is only meant to track changes since the last release, so I would omit this.
```suggestion
```